src/docs/reference/modules/Features/README.md

```diff
@@ -1,125 +1,3 @@
 # Features (`OrchardCore.Features`)
 
-List of features available in Orchard Core:
-
-| Name | Available |
```

Comment:
I don't think this section is useful. If somebody is interested in it the [roadmap](https://github.com/OrchardCMS/OrchardCore/wiki/Roadmap) is already linked. Otherwise, now there's more docs on all the features.